FloconDesktop/composeApp/src/commonMain/kotlin/io/github/openflocon/flocondesktop/core/domain/settings/StartAdbForwardUseCase.kt

```diff
@@ -1,6 +1,6 @@
 package io.github.openflocon.flocondesktop.core.domain.settings
 
-import io.github.openflocon.flocondesktop.SERVER_PORT
+import com.flocon.data.remote.server.SERVER_PORT
```

Comment:
This is a link domain -> data, should be the opposite,
Maybe move the port to a constant class inside domain